### PR TITLE
[zh] Sync Istio publishes results of 2022 security audit blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync Istio publishes results of 2022 security audit blog into Chinese:
```
content\zh\blog\2023\ada-logics-security-assessment\index.md
```

Note: please merge this translate after the PR: https://github.com/istio/istio.io/pull/13260 merged, because there is a link of that blog in this update.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/ada-logics-security-assessment/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Istio 的第一次安全评估中报告的所有 18 个问题都已被发现并得到修复。
+
+### 模糊测试{#fuzzing}
+
+[OSS-Fuzz 项目](https://google.github.io/oss-fuzz/)帮助开源项目执行免费的[模糊测试](https://en.wikipedia.org/wiki/Fuzzing)。
+Istio 已被集成到 OSS-Fuzz 中，有 63 个连续运行的模糊测试器：
+这种支持是[由 ADA Logics 和 Istio 团队于 2021 年底建立](https://adalogics.com/blog/fuzzing-istio-cve-CVE-2022-23635)。
+
+{{< quote >}}
+**“[我们]通过优先考虑 Istio 的安全关键部分来开始模糊测试评估。
```
Comment (windsonsea) [inline content/zh/blog/2023/ada-logics-security-assessment/index.md:113]:
```suggestion
**“[我们]通过优先考虑 Istio 的安全关键部分来开始模糊测试评估。
```
`[我们]` en 这里是不是漏加链接了？

**Review by windsonsea — COMMENTED** [review]
pick a nit

File: content/zh/blog/2023/ada-logics-security-assessment/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/blog/2023/ada-logics-security-assessment/index.md:113]: 感觉不太正常，需要确认一下。

Comment (wilsonwu) [reply content/zh/blog/2023/ada-logics-security-assessment/index.md:113]: I submit a issue to check this: https://github.com/istio/istio.io/issues/13263

**Review by windsonsea — APPROVED** [review]

**Pull request merged**